### ISSUE TITLE
Crash When Set Date To 1350!

### ISSUE THREAD

**Issue by aelahmar** [msarhan/ummalqura-calendar#14]
Calendar now = new UmmalquraCalendar(1350, UmmalquraCalendar.SHAABAN, 11);

**Comment by arvi** [comment]
Yep. I can confirm it crashes when date is less than 1400. I hope this can be fixed.

**Comment by aelahmar** [comment]
Any update on this issue?
